api/lucy-bot.ts

```diff
@@ -5,13 +5,9 @@ import { ExtraEditMessage } from 'telegraf/typings/telegram-types';
 
 const PROD_ENV = process.env.NODE_ENV === 'production';
 
-if (!PROD_ENV) {
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  require('dotenv').config();
-}
-
 const bot = new Telegraf(process.env.LUCY_BOT_TOKEN || '');
 
+bot.use(Telegraf.log());
```

Comment:
What does `Telegraph.log()` do exactly?

Reply:
It will provide a detailed log something like this which can be useful to locate issue
```json
{
  "update_id": 24312332,
  "message": {
    "message_id": 173,
    "from": {
      "id": 862301703,
      "is_bot": false,
      "first_name": "Target_X",
      "username": "Target_X",
      "language_code": "en"
    },
    "chat": {
      "id": -450132051,
      "title": "test",
      "type": "group",
      "all_members_are_administrators": true
    },
    "date": 1602702162,
    "text": "/DevTalks",
    "entities": [
      {
        "offset": 0,
        "length": 9,
        "type": "bot_command"
      }
    ]
  }
}
```

Reply:
Oh, okay. In vercel logs?

Reply:
yeah

Reply:
Cool!